packages/contracts/contracts/TellerV2Storage.sol

```diff
@@ -143,4 +143,9 @@ abstract contract TellerV2Storage_G3 is TellerV2Storage_G2 {
     ICollateralManager public collateralManager;
 }
 
-abstract contract TellerV2Storage is TellerV2Storage_G3 {}
+abstract contract TellerV2Storage_G4 is TellerV2Storage_G3 {
+    // BidId to payment cycle type (custom or monthly)
+    mapping(uint256 => PaymentCycleType) public bidPaymentCycleType;
```

Comment:
why not include this in the bid struct ?

Reply:
subgraph complications 🫤